api/README.md

```diff
@@ -9,7 +9,7 @@ audit history where required.
 
 # Table of contents
 
-<!-- To update table of contents run: `npm run-script updatetoc` -->
+<!-- To update table of contents run: `npx doctoc --github --maxlevel 2 README.md` -->
```

Comment:
Might be neat to make this an `npm` script in `package.json` (either in project root, or api-specific).

Reply:
done